static/js/main.js

```diff
@@ -0,0 +1,8 @@
+$(document).ready(function() {
+  $('.date-input').datepicker({
+    dateFormat: 'dd-mm-yy',
```

Comment:
why the `yy` here rather than four character year?

Reply:
I'm sticking to the format of the default for the jQuery datepicker which is "mm-dd-yy".

Reply:
cool, I assumed that this was then displayed to the user in that format, but sounds like it is for internal use only